Mixpanel/Mixpanel.m

```diff
@@ -191,6 +194,7 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
 
         self.decideResponseCached = NO;
         self.showSurveyOnActive = YES;
+        self.enableABTestDesigner = YES;
```

Comment:
Decision has been made to leave this functionality as is (enabled by default) to achieve feature parity with [mixpanel-android](github.com/mixpanel/mixpanel-android)